README.md

```diff
@@ -8,15 +8,15 @@ We’re building the next generation of secure location logging to preserve priv
 
 Location logs provide time-stamped records of where users have been, allowing them to share information with health officials accurately and quickly. This helps support contact tracing efforts to slow the spread of the virus.
 
-What’s truly special about Safe Paths, though, is its privacy protection. Data never leaves user's device without their password entry and explicit consent. The location log generated by Safe Paths cannot be accessed from outside the user’s device, meaning data transfer occurs only if the user chooses to share it with the researcher.
+What’s truly special about Safe Paths, though, is its privacy protection. Data never leaves a user's device without their password entry and explicit consent. The location log generated by Safe Paths cannot be accessed from outside the user’s device, meaning data transfer occurs only if the user chooses to share it with a researcher or health official.
 
 ----
 
 <img align="right" src="./assets/PreviewUI.png" data-canonical-src="./assets/PreviewUI.png"/>
 
 Safe Paths is a ‘privacy-first’ app that allows you to log your GPS trails on your own phone. The information is stored locally and never shared with anyone (not even with us or MIT) until you explicitly decide to manually export the data. The location log generated by Safe Paths cannot be accessed from outside the user’s device. Location information can be imported and exported by the user and used in other projects and applications.
 
-Safe Paths logs your device’s location once every five minutes and stores 28 days of data in under 100KB of space – less space than a single picture.  But what is truly exciting about Safe Paths is its privacy protection.
+Safe Paths logs your device’s location once every five minutes and stores 28 days of data in under 100KB of space–less space than a single picture.
```

Comment:
I think we can just remove this phrase: `-less less space than a single picture.`

Reply:
Agreed, made that edit.